drivers/misc/intel/comm_widget/intel_comm_widget_messages.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "intel_comm_widget.h"
+#include "pmc_interface.h"
+
+/*
+ * Report number of used HP-SRAM memory banks to the PMC, unit is 32 KB.
+ */
+int intel_comm_widget_pmc_send_ipc(const struct device *dev, uint16_t banks)
+{
+	if (!cw_upstream_ready())
+		return -EBUSY;
+
+	uint32_t iface = FIELD_PREP(CW_PMC_IPC_OP_CODE, CW_PMC_OPC_SRAM_CONFIG) |
+			 FIELD_PREP(CW_PMC_IPC_SRAM_USED_BANKS, banks) |
+			 CW_PMC_IPC_BUSY;
+
+	cw_sb_write(CW_PMC_DESTID_VALUE, 0, CW_PMC_MAILBOX3_INTERFACE_ADDRESS, iface);
+
+	cw_upstream_wait_for_sent();
```

Comment:
Is this function defined anywhere? Can't find it in the code. Does this compile?
If it is not can you please add dummy functions that do nothing so it compiles?

Reply:
Yes, it is defined as static inline in drivers\misc\intel\comm_widget\intel_comm_widget.h. I always compile own code before pushing it as PR...

Reply:
Good for you, just wanted to confirm :-)